src/lib/components/pages/participant/process-page/list-process-page.tsx

```diff
@@ -0,0 +1,34 @@
+import { IBooth } from ".";
+import { CheckCircle2Icon } from "lucide-react";
+
+export const ListProcess = (props: { listProcess: IBooth[] }) => {
+  const { listProcess } = props;
+  return (
+    <section className="w-full flex flex-col gap-3">
+      {listProcess.map((item, index: number) => {
```

Comment:
for semantic, you can wrap this map with tag "ul" and then change tag "div" to "li", so the browser will detect this component as list

ref: https://stackoverflow.com/questions/13416866/why-should-i-use-ul-li-in-my-html